### PR TITLE
changed how leaderboard looks

### PR DESCRIPTION
**Pull request by HaMoOoOd25**
- I think this looks more compact and easier on the eyes :)
Screenshot 2023-12-08 at 3 41 50 PM

### CODE REVIEW

File: pkg/bot/leaderboard.go [hunk 1]
```diff
@@ -74,15 +80,36 @@ func (bot *Bot) GenerateLeaderboardEmbed(guildId string) (*discordgo.MessageEmbe
...
-			Value: fmt.Sprintf("%d points\n%s", member.LocalScore, stars),
-		})
+		usernames += fmt.Sprintf("`%d` %s\n", i+1, username)
+		points += fmt.Sprintf("`%d`\n", member.LocalScore)
+		stars_m += stars + "\n"
+
+		// leaderboardEmbed.Fields = append(leaderboardEmbed.Fields, &discordgo.MessageEmbedField{
+		// 	Name:  fmt.Sprintf("%d. %s", i+1, username),
+		// 	Value: fmt.Sprintf("%d points\n%s", member.LocalScore, stars),
+		// })
```
Comment (jackharrhy) [inline pkg/bot/leaderboard.go:95]:
```suggestion
```
no need to keep around :skull: code, we have git

**Review by jackharrhy — CHANGES_REQUESTED** [review]

**Comment by nint8835** [comment]
I agree it looks better on PC, however unfortunately this design only really renders properly on screens that are wide enough - here's how it looks on mobile, for example
image

**Comment by HaMoOoOd25** [comment]
I forgot to check for that. I'll think of something different.

**Comment by nint8835** [comment]
> I forgot to check for that. I'll think of something different.

I'm currently looking into doing this using an image rather than embed, so if you can't find a quick answer I'd say don't bother putting in too much time

**Comment by HaMoOoOd25** [comment]
Fair enough, I'll close the pr.

**Pull request closed without merging**